addons/web/static/src/views/form/form_group/form_group.xml

```diff
@@ -39,15 +39,28 @@
 </t>
 
 <t t-name="web.Form.InnerGroup.ItemComponent" owl="1">
-    <div class="o_cell o_wrap_label flex-grow-1 flex-sm-grow-0 w-100 text-break text-900">
-        <t t-component="cell.Component" t-if="cell.isVisible" t-props="cell.props"/>
-    </div>
-    <div
+    <t t-set="isBoolean" t-value="cell.props.fieldInfo.FieldComponent.name === 'BooleanField'"/>
```

Comment:
Since `isBoolean` is used by the following `if`only, you can avoid defining a variable and use the `if` direclty.

```xml
<t t-if="cell.props.fieldInfo.FieldComponent.name === 'BooleanField'">
```